#goal definition
string frame
float32 table_height
float32 toolholder_size
---
#result definition
geometry_msgs/PoseArray blocks
turtlebot2i_marker_manipulation/BlockPoseArray colored_blocks
---
#feedback
